workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -52,6 +52,7 @@ PlayerSettings:
   m_StackTraceTypes: 010000000100000001000000010000000100000001000000
   iosShowActivityIndicatorOnLoading: -1
   androidShowActivityIndicatorOnLoading: -1
+  tizenShowActivityIndicatorOnLoading: -1
```

Comment:
There are many changes in this file which relate to tizen, might be better to revert this file.

Reply:
Included by accident!